spec/Section 7 -- Response.md

```diff
@@ -244,6 +238,19 @@ discouraged.
 }
 ```
 
+### Path
+
+:: A _path field_ allows for the association to a particular field in a GraphQL
+result. This field must be a list of path segments starting at the root of the
+response and ending with the field to be associated with. Path segments that
+represent fields must be strings, and path segments that represent list indices
+must be 0-indexed integers. If the path is associated to an aliased field, the
+path must use the aliased name, since it represents a path in the response, not
+in the request.
+
+When the _path field_ is present on an "Error result", it indicates the response
+field which experienced the error.
```

Comment:
I personally would use the word "identifies" here, it feels like it has more precision... But this opinion is very weakly held.

"path field" -> "path entry" should be made if the above suggestions are accepted though.

```suggestion
When the _path entry_ is present on an "Error result", it identifies the
response field which experienced the error.
```